drivers/power/battery_charger.c

```diff
@@ -388,6 +388,16 @@ static int bat_charger_ioctl(FAR struct file *filep, int cmd,
         }
         break;
 
+      case BATIOC_VOLTAGE_INFO:
+        {
+          FAR int *outvoltsp = (FAR int *)((uintptr_t)arg);
+          if (outvoltsp)
+            {
+              ret = dev->ops->voltage_info(dev, outvoltsp);
```

Comment:
what's different between BATIOC_GET_VOLTAGE and BATIOC_VOLTAGE_INFO?

Reply:
BATIOC_GET_VOLTAGE used to get voltage from setting register.
BATIOC_VOLTAGE_INFO used to get actual output voltage from other register. 
One is the theoretical value and the other is the actual value.
This is not a common usage,so removed it.